File: src/main/java/org/example/bankramenserver/domain/auth/dto/request/KakaoLoginRequest.java
package org.example.bankramenserver.domain.auth.dto.request;

import jakarta.validation.constraints.NotBlank;

public record KakaoLoginRequest(
        @NotBlank(message = "카카오 액세스 토큰은 필수 항목입니다.")
        String kakaoAccessToken
) {
}

File: src/main/kotlin/org/example/bankramenserver/domain/auth/dto/request/KakaoLoginRequest.kt
package org.example.bankramenserver.domain.auth.dto.request
import jakarta.validation.constraints.NotBlank
data class KakaoLoginRequest(@field:NotBlank(message = "카카오 액세스 토큰은 필수 항목입니다.") val kakaoAccessToken: String) { fun kakaoAccessToken() = kakaoAccessToken }
